src/client.rs

```diff
@@ -43,7 +271,44 @@ pub struct ResponseFuture {
     inner: proto::OpaqueStreamRef,
 }
 
-/// Build a client.
+/// Client connection factory, which can be used in order to configure the
+/// properties of the HTTP/2.0 client before it is created.
```

Comment:
tioli: _Builds client connections..._ (conjugation)

Reply:
I'm not sure what the proposed change is. Substituting "Client connection factory" w/ `Builds client connections"?

Reply:
ah, I see (based on prior comment).